<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyTeams</class>
 <widget class="QDialog" name="modifyTeams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="AddModifyTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>541</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="modDel">
    <attribute name="title">
     <string>Modify Teams</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>371</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="form">
      <item row="0" column="0">
       <widget class="QLabel" name="labels">
        <property name="text">
         <string>Team</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="teamsComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labels_3">
        <property name="text">
         <string>Stadium Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="stadiumLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labels_2">
        <property name="text">
         <string>Seating Capacity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="seatingspinBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="locationLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labels_4">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="roofComboBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="surfaceComboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Playing Surface</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labels_6">
        <property name="text">
         <string>Date Opened</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="datespinBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labels_5">
        <property name="text">
         <string>Distance to Center Field</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="distancespinBox"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Roof Type</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="typologyComboBox"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ballpark typology</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="modify">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="add">
    <attribute name="title">
     <string>Add Teams</string>
    </attribute>
    <widget class="QPushButton" name="addTeam">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add File</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>modifyTeams</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>modifyTeams</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
